### PR TITLE
Feature/2017 11 23 io t virtual reality

### PR DESCRIPTION
**Pull request by beele**

### CODE REVIEW

File: _posts/2017-12-20-Virtual-Reality.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+- CryEngine
+
+Think of these as sort of photoshop for software.
+We can leverage the editors to see what we're working on and make changes immediately.
+These tools are ideal for Virtual Reality.
+Both Unity and Unreal Engine have plugins available for Augmented Reality.
+
+The often more business targeted applications are often augmented reality enabled applications.
+Both Apple and Google have made big efforts to support this on their respective OSes and basically every smartphone with a camera is now AR enabled.
+Apple's ARKit and Google's ARCore are the respective Augmented reality APIs for their IOS and Android platforms.
```
Comment (yannickdeturck) [inline _posts/2017-12-20-Virtual-Reality.md]: Augmented Reality or augmented reality (currently it's one with a capital and one without a capital)
iOS

File: _posts/2017-12-20-Virtual-Reality.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+Since the inception of these two APIs the amount of AR apps has greatly increased, Google is even discontinuing it's Tango project in favor of ARCore.
```
Comment (yannickdeturck) [inline _posts/2017-12-20-Virtual-Reality.md]: its

Maybe also make Tango clickable for more info on that project? As not everyone will know about it (including me :-)).

File: _posts/2017-12-20-Virtual-Reality.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+There are many other SDKs available, many of which even have their own Unity plugin.
+Vuforia and ARToolkit seem to be the most popular ones.
+
+Microsoft created the Windows Mixed Reality framework for Mixed, Virtual or Augmented reality apps to run on windows 10 based computers.
```
Comment (yannickdeturck) [inline _posts/2017-12-20-Virtual-Reality.md]: Reality (also with an capital?)
Windows

File: _posts/2017-12-20-Virtual-Reality.md [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+
+
+## Business cases & Examples
+So, as software developers, what can we do with all this fancy tech?
+
+Our Dutch colleagues already made an awesomely detailed VR application to help train personnel of the royal Dutch navy.
```
Comment (yannickdeturck) [inline _posts/2017-12-20-Virtual-Reality.md]: Royal Dutch Navy (I think they all need to start with a capital)

File: _posts/2017-12-20-Virtual-Reality.md [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+
+For us at Ordina, we see most potential in AR and MR.
+Mobile based AR applications that can be used for showcases, in the IoT world, or even just augmented digital signage with inverse marker based AR.
+
+
+## Conclusion
+The years to come will bring even more advancements to the wonderful world of virtual, augmented and mixed realities.
+A lot of things are still shaping up and being developed, ever improving on earlier versions.
+A lot of solutions and standards are still being figured out.
+Which means now is a great time for us to try stuff out, and get a feel for the technology, whilst preparing and working on proof of concept applications.
```
Comment (yannickdeturck) [inline _posts/2017-12-20-Virtual-Reality.md]: ... means **that** ...

File: _posts/2017-12-20-Virtual-Reality.md [hunk 6]
```diff
@@ -0,0 +1,270 @@
+---
+layout: post
+authors: [michael_vandendriessche, kevin_van_den_abeele]
+title: 'An introduction to virtual and alternate reality'
+image: /img/virtualreality/banner.jpg
+tags: [Internet of Things, IoT, Virtual reality, Alternate reality, Mixed reality, Merged reality, Oculus rift, Oculus, HTC vive, vive, steamVR, steam, unreal engine, unity, hololens, holo lens, microsoft, google glass, google]
```
Comment (yannickdeturck) [inline _posts/2017-12-20-Virtual-Reality.md]: Capitalising all tags will look better.

File: _posts/2017-12-20-Virtual-Reality.md [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+- Unity
+- Unreal Engine
+- CryEngine
+
+Think of these as sort of photoshop for software.
+We can leverage the editors to see what we're working on and make changes immediately.
+These tools are ideal for Virtual Reality.
+Both Unity and Unreal Engine have plugins available for Augmented Reality.
+
+The often more business targeted applications are often augmented reality enabled applications.
```
Comment (Turbots) [inline _posts/2017-12-20-Virtual-Reality.md]: too many **often**

**Comment by Turbots** [comment]
general remark: emphasize some keywords or key phrases a bit more by using **double star notation**. Makes the text **even more readable**! :-)

**Pull request merged**